static/js/directives/youtubevideo.js

```diff
@@ -41,20 +102,63 @@ define(['jquery', 'underscore', 'text!partials/youtubevideo.html', 'bigscreen'],
 			var player = null;
 			var playerReady = null;
 			var isPaused = null;
-			var seekDetector = null;
 			var playReceivedNow = null;
-			var prevTime = null;
-			var prevNow = null;
 			var initialState = null;
+			var sandboxFrame = $("#youtubeplayer", $element)[0];
```

Comment:
ID's should never be used inside a directive. Please change to a class and use $element.find to retrieve the element.